go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify v1.4.0
 	github.com/timakin/bodyclose v0.0.0-20190930140734-f7f2e9bca95e
+	github.com/tommy-muehle/go-mnd v0.0.0-20190903201840-c93e405da530
```

Comment:
Can we change this to reference a tagged release version?

Reply:
I tried before, but it gives me the below error (even the tag is available)

Reply:
oh I got it now, the tag in github.com/tommy-muehle/go-mnd is not prefixed with `v`, so we can only use pseudo-versions only

Reply:
raised one issue here https://github.com/tommy-muehle/go-mnd/issues/1